bindings/cpu_profiler.cc

```diff
@@ -108,7 +108,12 @@ class MeasurementsTicker {
 
   ~MeasurementsTicker() {
     uv_timer_stop(&timer);
-    uv_close(reinterpret_cast<uv_handle_t *>(&timer), nullptr);
+
+    auto handle = reinterpret_cast<uv_handle_t *>(&timer);
+
+    if (uv_is_active(handle)) {
```

Comment:
Can you add a comment to here? Why are we doing this?

Also:

```suggestion
    if (uv_is_active(handle)) {
```

Reply:
@anonrig ofc, yes!